src/main/java/mate/academy/bookshop/model/order/Order.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.bookshop.model.order;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.NamedAttributeNode;
+import jakarta.persistence.NamedEntityGraph;
+import jakarta.persistence.NamedSubgraph;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import mate.academy.bookshop.model.User;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode(exclude = {"user", "status", "orderItems"})
+@ToString(exclude = {"user", "status", "orderItems"})
+@NamedEntityGraph(
+        name = "orderWithItemsAndBooks",
+        attributeNodes = {
+                @NamedAttributeNode(value = "user"),
+                @NamedAttributeNode(value = "orderItems", subgraph = "orderItemsGraph")
+        },
+        subgraphs = {
+                @NamedSubgraph(
+                        name = "orderItemsGraph",
+                        attributeNodes = @NamedAttributeNode(value = "book")
+                )
+        }
+)
```

Comment:
is it necessary?

Reply:
i do this to fetch all nesessary fields from db, to avoid lazy initialization exception